students/pkoleyni/lesson04/mailroom_2.py

```diff
@@ -174,7 +176,4 @@ def menu_selection():
 
 
 if __name__ == '__main__':
-    # updating total donation made by each donor and add it to the dictionary
-    for name in list_donors():
-        update_total_donation_to_dic(name)
```

Comment:
remember to change this line `if main_menu_dic[response]() == "exit menu":` to `main_menu_dic[response]()`

since you no longer need to check for "exit menu"

you can address this as part of lesson5